braindecode/datasets/base.py

```diff
@@ -108,6 +107,12 @@ def set_description(self, description, overwrite=False):
                 self._description.pop(key)
         self._description = pd.concat([self.description, description])
 
+    def save(self, save_dir, overwrite=False):
+        """See braindecode.datautil.serialization.save_base_dataset.
+        """
+        from braindecode.datautil.serialization import save_base_dataset
```

Comment:
Please move saving code here.

Reply:
Ah, no. Please remove `save_base_dataset`. We don't need it anymore. We save on `BaseConcatDataset` level.